src/TimeService/Startup.cs

```diff
@@ -89,11 +99,11 @@ private void ConfigureRepositories(IServiceCollection services)
             services.AddTransient<IWorkTimeRepository, WorkTimeRepository>();
         }
 
-        public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
+        public void Configure(IApplicationBuilder app, ILoggerFactory loggerFactory)
         {
             app.UseHealthChecks("/api/healthcheck");
```

Comment:
just a question, should we leave this line? because in others pull requests we deleted it

Reply:
Fixed.